third_party/airflow/armada/operators/armada.py

```diff
@@ -33,29 +30,25 @@
 )
 from armada.jobservice import jobservice_pb2
 
-
 armada_logger = logging.getLogger("airflow.task")
 
 
 class ArmadaOperator(BaseOperator):
     """
-    Implementation of an ArmadaOperator for airflow.
-
+    Implementation of an ArmadaOperator for Airflow.
     Airflow operators inherit from BaseOperator.
 
-    :param name: The name of the airflow task
+    :param name: The name of the Airflow task
```

Comment:
With all these changes you will have to generate the documentation if you want to keep this change.  I’d normally suggest against making changes like this as it distracts the reviewer from the topic at hand.

Reply:
ok thank you @kannon92